src/router/router.types.ts

```diff
@@ -33,13 +33,13 @@ export type Navigate = <T extends RouteKey>(
 ) => void
 
 export type RouteAuthLevel = 'any' | Readonly<Array<UserProductRole>>
-export type RouteConfig = {
+
+export type LocalizedRoute = {
   PATH: Record<LangCode, Readonly<string>>
   LABEL: Record<LangCode, Readonly<string>>
-  EXACT?: boolean
   COMPONENT: React.FC
   PUBLIC: boolean
-  AUTH_LEVELS?: RouteAuthLevel
+  AUTH_LEVELS: RouteAuthLevel
```

Comment:
@Carminepo2 let's remove `any` as a possible value, always list all the `AUTH_LEVELS` allowed to see the content (`any === ['admin', 'api', 'security']`